### ISSUE TITLE
Create a modernized accessibility page

### ISSUE THREAD

**Issue by DanielleThierryUSDSVA** [department-of-veterans-affairs/va.gov-team#49797]
### Veteran story

I need clear and accessible content that explains VA's accessibility policy and provides an easy way to provide feedback on accessibility issues.

### VA story

We need to comply with laws and regulations on accessibility statements and feedback mechanisms.

## What's needed

We need to create a new modernized page that our footer can link to that provides a clear explanation of VA's accessibility policy and provides an easy way to provide feedback.

### Tasks to complete

- [x] Create draft content (Josh and team)
- [x] Determine IA approach and URL (Mikki, Joni)
- [x] Review content (Danielle)
- [x] Build Drupal page (Danielle/content team)
- [x] Get early feedback from 508 office (Josh, Martha)
- [x] Make any needed edits (Danielle, Josh)
- [x] Copy edit and review against pre-publishing checklist (Megan)
- [x] Get SME approval from 508 office (Martha, Josh)
- [x] Get additional legal reviews and any other SME approvals (Danielle)
- [x] Make any needed edits (Danielle, Josh)
- [x] Staging review (platform governance team)
- [ ] Final pre-publishing review and publish (Megan)
- [ ] Update footer link to point to new URL (content team/public websites team)

**Comment by DanielleThierryUSDSVA** [comment]
@RLHecht Going to prioritize this work because the current TeamSite page with accessibility information and a way to provide feedback on accessibility is not accessible. I'll work directly with the team on this one and then ask Megan to copy edit and review pre-publishing. Thanks!

**Comment by mnorthuis** [comment]
@DanielleThierryUSDSVA can you add a link to the existing page in the original description for reference?

**Comment by artsymartha68** [comment]
The "Accessibility" footer link currently points to https://www.section508.va.gov/

**Comment by DanielleThierryUSDSVA** [comment]
Thanks, @artsymartha68! 

 @mnorthuis @heyjoni Here's the [draft Drupal page](http://preview-prod.vfs.va.gov/preview?nodeId=51264) so you have the draft content to consider when determining IA.

**Comment by heyjoni** [comment]
Thanks for sharing the draft, Danielle. Per our Slack convo, we will replace the "Accessibility" footer link so that it points to the new page, which will have the URL va.gov/accessibility-at-va. We will no longer link to https://www.section508.va.gov from the footer since the new page will refer to it. Since the new page will be root-level, it won't be in the left nav. 

Here's the [detailed IA guidance](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/information-architecture/ia-reviews/accessibility.md) for this page.

**Comment by joshkimux** [comment]
@DanielleThierryUSDSVA Confirming that the Section 508 office (Christopher Walker) approved the current draft for publication ✨ 

They'll be looking to provide more feedback once it is launched. We agreed that this would be a MVP with future iterations as we learn more about disabled Veteran needs.

**Comment by joshkimux** [comment]
@megzehn this is ready for you to copy edit and review against the pre-publishing checklist ✨

**Comment by DanielleThierryUSDSVA** [comment]
That's great news, @joshkimux! My legal contact in ORMDI who worked with me on the civil rights page just connected me with legal reviewers in the Disability Program office who can provide the further legal review we wanted to get. So I'll be sending the links to them. I'm also checking on any OPIA approvals we might need since this is the footer.

@megzehn This is still good to copy edit and check against our checklist, but please hold on publishing until we get any further needed approvals. Thanks!

**Comment by DanielleThierryUSDSVA** [comment]
Update: No OPIA approval needed, just an awareness note that I can send before we publish.

**Comment by DanielleThierryUSDSVA** [comment]
Hi @RLHecht. Could you schedule a staging review for this page for  late next week? Thank you!

**Comment by megzehn** [comment]
Hello! Confirming that I copy edited this page and left in drafts for now. CC @DanielleThierryUSDSVA and @joshkimux. 

Thank you!

**Comment by DanielleThierryUSDSVA** [comment]
Thank you so much, @megzehn! 

I've asked our final reviewers for any questions or concerns by Monday, 12/12. 

One question came in so far about adding a link to the reasonable accommodations for employees page somewhere in this flow of information. I added it as a related link to the [civil rights and discrimination page](https://www.va.gov/resources/your-civil-rights-and-how-to-file-a-discrimination-complaint/) since we link there for info about reasonable accommodations and discrimination information in general. @megzehn whenever you have a moment, I'd appreciate it if you could make sure I didn't introduce any typos in that related link and then we can publish that update to that page anytime.

**Comment by megzehn** [comment]
Looks good, @DanielleThierryUSDSVA! Confirming that I republished the [civil rights and discrimination page](https://www.va.gov/resources/your-civil-rights-and-how-to-file-a-discrimination-complaint/) with your added link.

**Comment by DanielleThierryUSDSVA** [comment]
Confirming that the Disability program office and the Chief of reasonable accommodations have approved this content with no changes. So we're good to launch after staging review.

**Comment by DanielleThierryUSDSVA** [comment]
Hi @megzehn. This page is now ready for the final pre-publication checklist and publishing at the URL specified by Joni. 

@RLHecht As soon as this page is published, we can go ahead and update the link in the footer (w/ PW help).

**Comment by RLHecht** [comment]
Update: I created a ticket for PW to update the link in the footer. This may not be done before code freeze but we can still publish the page now. Thanks!

**Comment by megzehn** [comment]
Thank you, @RLHecht and @DanielleThierryUSDSVA ! Confirming that I went through the pre-pub checklist and published this today. 

https://www.va.gov/accessibility-at-va

**Comment by RLHecht** [comment]
This work is complete. Closing ticket.
